specs/protocol/configurability.md

```diff
@@ -0,0 +1,71 @@
+| Challenge Period                      | Length of time for which an output root can be removed, and for which it is not considered finalized.                                                                                                                                                             |
+| Compression ratio                     | How much compression the batch submitter applies to batches before submission (see [Channel](../glossary.md#channel)).       |
+| Data Availability                     | Where L2 transaction calldata is posted.                                                                                     |
+| Fault Proof Window / Challenge Window | Duration network participants have to challenge the integrity of an output root.                                             |
+| Fee margin                            | Markup on transactions compared to the raw L1 data cost.                                                                     |
+| Fee vault contracts                   | Contracts which dictate how user fees are distributed.                                                                       |
+| Gas limit                             | L2 block gas limit.                                                                                                          |
+| Gas Token                             | Native token used to pay for gas.                                                                                            |
+| Genesis state                         | Initial state at chain genesis, including code and storage of predeploys (all L2 smart contracts). See [Predeploy](../glossary.md#l2-genesis-block).                                            |
+| L1 smart contracts                    | The chain’s L1 smart contracts.                                                                                              |
+| L2 block time                         | Frequency with which blocks are produced as a result of derivation.                                                          |
+| Output frequency                      | Frequency with which output roots are submitted to L1.                                                                       |
+| Resource config                       | Config for the EIP-1559 based curve used for the deposit gas market.                                                         |
+| Sequencing window                     | Maximum allowed batch submission frequency, after which L1 fallback is triggered in derivation.                                                                                                                                                            |
+| Start block                           | Block at which op-node starts searching for logs from.                                                                       |
```

Comment:
This is better defined as the block at which the system config was initialized the first time